packages/webdriverio/src/commands/browser/reloadSession.ts

```diff
@@ -41,6 +41,9 @@ export default async function reloadSession (this: WebdriverIO.Browser) {
         log.warn(`Suppressing error closing the session: ${err.stack}`)
     }
 
+    if (this.puppeteer)
+        this.puppeteer = undefined;
```

Comment:
```suggestion
    if (this.puppeteer) {
        delete this.puppeteer
    }
```

Reply:
That does not work for some reason. I get this error on the 2nd `browser.url` call, first one goes through without any issues. 

![image](https://user-images.githubusercontent.com/10925588/119223274-540d3080-baf0-11eb-8d61-35d7787ca030.png)

This is a sample project I am running by linking the package.

**Stacktrace**:
```javascript
[0-0] Error in "1: I search for 'reloadSession'"
invalid session id: No active session with ID 78bdf25b0b241b231f6b3fec05a7b5b5
    at Object.getErrorFromResponseBody (C:\Users\jonne\workspace\node.js\wdio-jonn\packages\webdriver\src\utils.ts:221:12)
    at WebDriverRequest._request (C:\Users\jonne\workspace\node.js\wdio-jonn\packages\webdriver\src\request.ts:195:23)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at Browser.runCommandWithHooks (C:\Users\jonne\workspace\node.js\wdio-jonn\packages\wdio-sync\src\wrapCommand.ts:125:25)
    at Browser.runCommandWithHooks (C:\Users\jonne\workspace\node.js\wdio-jonn\packages\wdio-sync\src\wrapCommand.ts:118:24)
    at Browser.wrapCommandFn (C:\Users\jonne\workspace\node.js\wdio-jonn\packages\wdio-sync\src\wrapCommand.ts:78:44)
    at World.<anonymous> (C:\Users\jonne\workspace\node.js\browser.mock-defect\features\step-definitions\steps.js:7:13)
    at World.executeSync (C:\Users\jonne\workspace\node.js\browser.mock-defect\node_modules\@wdio\sync\build\index.js:38:22)
    at C:\Users\jonne\workspace\node.js\browser.mock-defect\node_modules\@wdio\sync\build\index.js:69:68

```

Reply:
@christian-bromann What next, do you want to try something else?